cpp/src/Ice/Exception.cpp

```diff
@@ -15,7 +15,7 @@
 // For UINTPTR_MAX on Ubuntu Precise
 //
 #ifndef __STDC_LIMIT_MACROS
-#    define __STDC_LIMIT_MACROS
+#    define __STDC_LIMIT_MACROS // NOLINT
```

Comment:
Ditto. What doesn't the linter like here?

Reply:
It doesn't like that we define a __XXX symbol.